collective/trustedimports/soap.rst

```diff
@@ -0,0 +1,61 @@
+SOAP
+===
+
+When used in RestrictedPython we can still use Zeep
+
+>>> teval("from zeep import Client;return Client('http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL')")
+<zeep.client.Client ... at ...>
+
+>>> teval("from zeep.exceptions import Error;raise Error('Test error')")
+Traceback (most recent call last):
+...
+Error: Test error
+
+
+We cann't set transport option for WSDL service
+
+>>> teval("from zeep import Client;client = Client('http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL', settings = {'force_https':True})")
+Traceback (most recent call last):
+...
+ValueError: Argument(s) 'wsdl,settings' have values not supported in a restricted python call
+
+>>> teval("from zeep import Client;client = Client('http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL', settings = {'extra_http_headers':''})")
+Traceback (most recent call last):
+...
+ValueError: Argument(s) 'wsdl,settings' have values not supported in a restricted python call
+
+We cann't open connection to URL in blacklist
```

Comment:
set the blacklist env here. make sure you remove it after

Reply:
@djay , I move the env code into doctest